### PR TITLE
Initial implementation of frakti

### PR DESCRIPTION
**Pull request by feiskyer**
Initial impl of frakti. This PR

- Listen frakti at local socket /var/run/frakti.sock
- Plugin framework of frakti
- Hyper as a plugin of frakti

### CODE REVIEW

File: manager/manager.go [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+	err := s.runtimeService.StartContainer(req.GetContainerId())
+	if err != nil {
+		glog.Errorf("StartContainer from runtime service failed: %v", err)
+		return nil, err
+	}
+
+	return &kubeapi.StartContainerResponse{}, nil
+}
+
+// StopContainer stops a running container with a grace period (i.e., timeout).
```
Comment (resouer) [inline manager/manager.go]: nit: `i.e., timeout` -> `i.e. timeout`

File: runtime/hyper/hyper.go [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+func (h *Runtime) CreateContainer(podSandBoxID string, config *kubeapi.ContainerConfig, sandboxConfig *kubeapi.PodSandboxConfig) (string, error) {
+	return "", fmt.Errorf("Not implemented")
+}
+
+// StartContainer starts the container.
+func (h *Runtime) StartContainer(rawContainerID string) error {
+	return fmt.Errorf("Not implemented")
+}
+
+// StopContainer stops a running container with a grace period (i.e., timeout).
```
Comment (resouer) [inline runtime/hyper/hyper.go]: same nit

File: manager/manager.go [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+	}
+
+	return &kubeapi.PodSandboxStatusResponse{Status: podStatus}, nil
+}
+
+// ListPodSandbox returns a list of SandBox.
+func (s *FraktiManager) ListPodSandbox(ctx context.Context, req *kubeapi.ListPodSandboxRequest) (*kubeapi.ListPodSandboxResponse, error) {
+	glog.V(3).Infof("ListPodSandbox with request %s", req.String())
+
+	items, err := s.runtimeService.ListPodSandbox(req.Filter)
```
Comment (resouer) [inline manager/manager.go]: Is it better to use `GetFilter()`, or it makes no difference?

**Comment by resouer** [comment]
lgtm except for some nits

**Comment by feiskyer** [comment]
@dchen1107 Please help to add bots and enable CI for this project.

File: manager/manager.go (hunk 3 above)
Comment (feiskyer) [inline manager/manager.go]: @resouer GetFilter() actually returns the req.Filter.

File: manager/manager.go (hunk 1 above)
Comment (feiskyer) [inline manager/manager.go]: done.

File: runtime/hyper/hyper.go (hunk 2 above)
Comment (feiskyer) [inline runtime/hyper/hyper.go]: done.

**Comment by resouer** [comment]
@feiskyer  This PR lgtm, should I merge this or wait for adding CI workflow?

**Pull request merged**